### DISCUSSION TITLE
请问是否支持Details Panel Customizations？

### DISCUSSION THREAD

**Discussion by Lance3d** [cgerchenhp/UE_TAPython_Plugin_Release#27] [General]
请问能不能用TAPython比较快速的实现Details Panel Customizations？即不用写cpp代码就可以修改Details里面的信息，谢谢！
https://docs.unrealengine.com/5.3/en-US/details-panel-customizations-in-unreal-engine/

**Comment by cgerchenhp** [comment]
目前没有直接修改和添加Details Panel的功能，有些简单的修改可以用类似下面这个hack的方式做：https://gist.github.com/cgerchenhp/60fe6b83b46371fee308f4df1ecf4328

但也仅限简单需求修改，这个做法是需要使用这不停查询和修改状态的
![G018_ColorNestedAttributes](https://github.com/cgerchenhp/UE_TAPython_Plugin_Release/assets/1730497/a4642c93-1014-4b33-898c-478d722aadb9)

**Reply by Lance3d** [reply]
感谢大佬的及时回复
